source/administration/config-settings.rst

```diff
@@ -3515,6 +3506,18 @@ This setting defines the frequency of cluster request time logging for :doc:`../
 | This feature's ``config.json`` setting is ``"ClusterLogTimeoutMilliseconds": 2000`` with numerical input.                                                            |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Read Only Config
+^^^^^^^^^^^^^^^^
+**True**: Changes made to settings in the System Console are ignored.
+
+**False**: Changes made to settings in the System Console are written to ``config.json``.
```

Comment:
The reference here to `config.json` is the kind of change we may want to holistically update in the future to decouple ourselves from the file-based orientation.

Reply:
Makes sense and there are likely many more throughout the docs.  Propose tackling that effort separately.